make.cmd

```diff
@@ -61,12 +61,12 @@ Test\test-ipy-tc.cmd /category:Languages\IronPython\IronPython\2.X
 goto :exit
 
 :distclean
-msbuild /t:DistClean /p:BuildFlavour=Release /verbosity:minimal /nologo
-msbuild /t:DistClean /p:BuildFlavour=Debug /verbosity:minimal /nologo
+msbuild /t:DistClean /p:BuildFlavour=Release /verbosity:minimal /nologo /p:Platform="Any CPU"
+msbuild /t:DistClean /p:BuildFlavour=Debug /verbosity:minimal /nologo /p:Platform="Any CPU"
 goto :main
 
 :main
-msbuild Build.proj /t:%_target% /p:BuildFlavour=%_flavour% /verbosity:minimal /nologo
+msbuild Build.proj /t:%_target% /p:BuildFlavour=%_flavour% /verbosity:minimal /nologo /p:Platform="Any CPU" /bl:build.binlog
```

Comment:
Maybe we should do different binlogs for release and debug? `/bl:bin/%_flavour%/build.binlog` or `/bl:build-%_flavour%.binlog`?